lib/scope.js

```diff
@@ -2082,12 +2118,16 @@ module.exports = {
     * @param ensure_all_vars_used {bool} [false] = Default false. If true, errors if any
     *     object in var_data is not used during running of this function.
     */
-    if ( session_vars.get_debug() ) { console.log( `~~~~~~~~~~~~~~ scope.setFields() ~~~~~~~~~~~~~~` ); }
+    log.debug({
+      code: `ALK0139`,
+      type: `info`,
+      pre: `~~~~~~~~~~~~~~ scope.setFields() ~~~~~~~~~~~~~~`
+    });
```

Comment:
We added `log.console()` for plain old pretty stuff and we should just use that here.

Reply:
That function was silly. This and related stuff fixed.